.github/PROCESS.md

```diff
@@ -64,35 +64,22 @@ Try to estimate on the planning as best as you can.
    not just for us.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not mention the issue number but describe the fixed issue instead.
-3. Most of the time pull requests are related to some issues.
-   In such cases, pull request **description** should mention the issue number
-   like `Closes #{issue_number}.` or `Closes org/repo#{issue_number}.`.
-   Do not use URLs like `https://github.com/org/repo/issue/123` or paths like `org/repo/issue/123`
-   even if they are rendered the same on GitHub.
-   If you propose a tiny fix, there is no needed to create a new issue.
-4. We provide a pull requests template that includes suggestions and readiness checklist.
+3. We provide a pull requests template that includes suggestions and readiness checklist.
    Please use it.
-5. It is fine to send several sequential pull requests for one issue to make them easier to review.
+4. It is fine to send several sequential pull requests for one issue to make them easier to review.
    In that case, please still use the `Closes` word as described above
    (because [words like "refs" do not link PR to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)),
    but don't forget to reopen the issue once PR is merged,
    but the issue as a whole is not done.
-6. Please create a draft pull request as soon as you start working on an issue; that is needed for our process tooling.
+5. Please create a draft pull request as soon as you start working on an issue; that is needed for our process tooling.
    Once it is ready for review, please mark it as such.
    After that, there is no need to switch between draft and non-draft states.
-7. In the pull request review conversations, please either leave a new comment or resolve (close) the conversation,
-   but do not do that simultaneously.
-   That ensures that the other person can read all comments.
-8. Please don't forget to click "re-request review" buttons once PR is ready for re-review.
-9. Commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
-   All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
-   Autogenerated GitHub's squash commit's body should be manually replaced by "Closes #{issue_number}.".
-   Do not leave original commit messages in the squashed commit body.
-10. Pull requests should be merged by an auto-merge that the author should enable.
+6. Please don't forget to click "re-request review" buttons once PR is ready for re-review.
```

Comment:
Please move that point to the CONTRIBUTING.md too. And then do it once PR is ready for re-review :)

Reply:
Oh I mixed it with draft things, will move it back to CONTRIBUTING.md